bitrise.yml

```diff
@@ -0,0 +1,90 @@
+format_version: 4
+default_step_lib_source: https://github.com/bitrise-steplib/bitrise-step-unity-install.git
+
+app:
+  envs:
+  # If you want to share this step into a StepLib
+  - BITRISE_STEP_ID: unity
+  - BITRISE_STEP_VERSION: 0.0.1
+  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/bitrise-steplib/bitrise-step-unity-install.git
+  - MY_STEPLIB_REPO_FORK_GIT_URL: $MY_STEPLIB_REPO_FORK_GIT_URL
+
+workflows:
+  test:
```

Comment:
If this can be run on any (?) Stack it coukld be added as an e2e test, see xcode-test repo for an example: https://github.com/bitrise-steplib/steps-xcode-test/blob/92d41f39d5b4e6a58f56b77977103c8af0da281d/bitrise.yml#L13. Here the test are in a different bitrise.yml: https://github.com/bitrise-steplib/steps-xcode-test/blob/master/e2e/bitrise.yml, and they are automagically ran on multiple types of Stacks